configurecompiler.cmake

```diff
@@ -446,14 +446,25 @@ if (CLR_CMAKE_PLATFORM_UNIX)
     add_compile_options(-Werror)
   endif(CLR_CMAKE_WARNINGS_ARE_ERRORS)
 
+  # Disabled common warnings
+  add_compile_options(-Wno-unused-variable)
+  add_compile_options(-Wno-unused-value)
+  add_compile_options(-Wno-unused-function)
+
+  #These seem to indicate real issues
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-invalid-offsetof")
+
+  # The following warning indicates that an attribute __attribute__((__ms_struct__)) was applied
```

Comment:
This comment was was incorrectly moved, it applies to the `-Wno-incompatible-ms-struct`

Reply:
done